docs/v6-migration.md

```diff
@@ -200,6 +202,12 @@ The series or movie _must be added in your instance of Sonarr or Radarr._ You **
 **We do not query any external metadata servers.**
 :::
 
+#### Ensemble or "Torrent Aggregation"
+
+Many of you may be familiar with things like [seasonpackarr](https://github.com/nuxencs/seasonpackarr) which - for qBittorrent - will aggregate/join your already downloaded episodes into season packs when they are combined and uploaded on your trackers, and inject them. `cross-seed` can now do the same, and potentially more, linking and then creating a "combined" set of matching torrents or data to get you seeding data you already have faster. This applies to things like season packs, but also could be used in other cases like anime "seasons" or multi-season packs.
+
+This functionality for season packs is linked to the new option, `seasonFromEpisodes` - which can be set to either `true` or `false`.

```

Comment:
```suggestion
Many of you may be familiar with things like [seasonpackarr](https://github.com/nuxencs/seasonpackarr) which - for qBittorrent - will aggregate/join your already downloaded episodes into season packs when they are combined and uploaded on your trackers, and inject them. `cross-seed` can now do the same, linking and then creating a "combined" set of matching torrents or data to get you seeding data you already have faster. Currently, only season packs from individual episodes is supported but we may support additional aggregation in the future.

This functionality for season packs is linked to the new option, `seasonFromEpisodes` - which can be set to either `true` or `false`.

:::tip
This feature works best with `matchMode: "partial"` and [Sonarr](./basics/options.md#sonarr).
:::
```

Reply:
Maybe the 'for qbittorrent' line is a bit misleading that this also only works for qbittorrent?

Reply:
Yeah it probably isn't necessary. Though I think it should be clear it doesn't apply to cross-seed.